test/src/901-printchangeset/main

```diff
@@ -0,0 +1,128 @@
+cvmfs_test_name="PrintChangesetNotice prints"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="quick"
+
+cvmfs_run_test() {
+  logfile=$1
+  ADD_COUNTER=0
```

Comment:
--> `local add_counter=0`.  The all caps variables indicate global scope, and we do want to use local variables if we can.